src/Components/UI/Carousel/Carousel.jsx

```diff
@@ -21,12 +24,18 @@ const Carousel = ({ brandData }) => {
   useEffect(() => {
     const interval = setInterval(() => {
       nextSlide();
-    }, 2000);
+    }, 5000);
     return () => clearInterval(interval);
   }, [currentIndex]);
 
   return (
-    <section className="carousel">
+    <section
+      className="carousel"
+      onClick={() => {
+        console.log("clicked");
+        navigate("/products");
+      }}
+    >
       <LeftArrow className="arrow arrow--left" onClick={prevSlide} />
```

Comment:
In ui the buttons are navigating to the products page because of the section onclick of carousel component.